src/app/models/Build.ts

```diff
@@ -0,0 +1,13 @@
+import { Revision } from './Revision';
+import { BuildWorker } from './BuildWorker';
+
+export interface Build {
+  id: string;
+  project_id: string;
+  type: string;
+  provider: string;
+  revision: Revision;
+  payload?: string;
+  script?: string;
+  worker?: BuildWorker;
```

Comment:
As an aside these properties were "optional" in that they're not always returned by the API had me running in circles for awhile.